doc/rls_mode/linting.md

```diff
@@ -0,0 +1,15 @@
+# RLS Mode Linting Page
+
+`RLS` checks the project and shows diagnostics while you are typing.
+
+You can see diagnostics in the Problems Panel.
+
+You can hover on a diagnostics to see what the diagnostic is.
+
+Executing a cargo command doesn't show any diagnostics.
+
+It is intentional decision.
+
+The reason is that there is no pretty way to hide a diagnostic after the diagnostic's cause is fixed.
+
+It may confuse people.

```

Comment:
What may confuse people? The old diagnostics after a fix?
`This would be confusing, hence the decision.`